### DISCUSSION TITLE
副本数量如何动态调整？

### DISCUSSION THREAD

**Discussion by gy7779** [milvus-io/milvus#23467] [Q&A and General discussion] [answered]
loadCollection 中可以指定加载的副本数：

LoadCollectionParam.newBuilder()
                      .withCollectionName("demo")
                      **.withReplicaNumber(2)**

当我想动态变更副本数时，重复调用了此接口，接口异常如下，提示我需要先释放后才可以变更

collection with different replica number 2 existed, release this collection first before changing its replica number[LoadParameterMismatched

我目前没有找到其他可以动态变更副本数的方法，请问 Milvus 可有对此操作的支持？

**Comment by xiaofan-luan** [comment, marked as answer]
目前还没有支持，需要先调用release再重新加载。
不过之后的版本会支持
